resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/k8s/KubernetesClusterSchedulerBackend.scala

```diff
@@ -47,7 +48,7 @@ private[spark] class KubernetesClusterSchedulerBackend(
     ExecutionContext.fromExecutorService(requestExecutorsService)
 
   protected override val minRegisteredRatio =
-    if (conf.getOption("spark.scheduler.minRegisteredResourcesRatio").isEmpty) {
+    if (conf.get(SCHEDULER_MIN_REGISTERED_RESOURCES_RATIO).isEmpty) {
```

Comment:
`getOrElse`?

Reply:
Because this is very interesting code that does not use it value even if `SCHEDULER_MIN_REGISTERED_RESOURCES_RATIO` is declared, we cannot use `getOrElse`. I am not sure why it is.

This code just checks whether `SCHEDULER_MIN_REGISTERED_RESOURCES_RATIO` is declared or not.